src/main.js

```diff
@@ -117,9 +126,7 @@ class PercyClient {
       interval: 50,
       max_tries: 5,
       throw_original: true,
-      predicate: function(err) {
-        return err.statusCode >= 500 && err.statusCode < 600;
-      },
+      predicate: retryPredicate,
```

Comment:
I'm guessing this codebase does the hanging commas too? Hm, we need a consistent lint guide :p